### DISCUSSION TITLE
WP Dashboard for the plugin add sort for City

### DISCUSSION THREAD

**Discussion by dougie-wougie** [code4recovery/12-step-meeting-list#1186] [Ideas]
On the five columns displayed in the meetings from the dashboard. Four can be sorted. Can a sort be added to "City?"

**Comment by mattbylett** [comment]
Thanks Dougle - There's a bit going on right now and I just wanted to acknowledge we've received this request and I'll take it back to the team for feedback.  I'm just new so not 100% sure of the process for this.  We'll get back to you

**Comment by brianw-area24** [comment]
On the site I just looked at, all 5 columns sort, the last column is "Region" which is the city on this site. What site are you looking at so we can take a better look at what might be happening?
Brian W

**Reply by dougie-wougie** [reply]
Thank you for your service. The Column named "City" is not sortable. 

d
![CleanShot 2023-08-10 at 05 06 08](https://github.com/code4recovery/12-step-meeting-list/assets/58369176/53339822-dd17-460a-a4b8-c16dd1913945)

**Comment by mattbylett** [comment]
Gotcha - Looking into it Dougie - thanks for your patience :)
